src/custom-elements/src/constants/classes.ts

```diff
@@ -90,6 +90,13 @@ export const CAROUSEL_CLASSES = {
 };
 //#endregion
 
+//#region Stat
+export const STAT_CLASSES = {
+  CAROUSEL: 'chi-stat',
+  ITEM: 'chi-stat__item',
```

Comment:
I would include this inside `CAROUSEL_CLASSES` as the **Stat** component is based on the carousel component and is not a standalone component like the carousel in this case. What do you think of this @jllr ?

`export const CAROUSEL_CLASSES = {
     ...,
     STAT_ITEM: 'chi-stat__item',
     ...
}`

Reply:
@KrzysztofDrozdowski @josecarlosgonzalezv I would move this class to Stat classes. Stat can work as a standalone component and is not fully dependant on carousel.

Reply:
@jllr @KrzysztofDrozdowski @josecarlosgonzalezv although this can be a valid implementation, knowing that the Stat is an independent component, and that maybe there is a WebComponent made of it in the future, I'd keep the classes separated as well.

Reply:
It's fixed 😊